src/js/fc.js

```diff
@@ -77,6 +77,7 @@ var FC = {
             uid:                              [0, 0, 0],
             accelerometerTrims:               [0, 0],
             name:                             '',
+            displayName:                      '',
```

Comment:
This can just be preallocated with a constant text for now.
```suggestion
            displayName:                      'JOE PILOT',
```